stepler/nova/steps/servers.py

```diff
@@ -141,7 +141,9 @@ def attach_floating_ip(self, server, floating_ip, check=True):
 
         if check:
             floating_ips = self.get_ips(server, 'floating').keys()
-            floating_ip.ip in floating_ips
+            assert floating_ip.ip in floating_ips, (
```

Comment:
@sergeychipiga, from my point of view just `assert` is more simple to use. And custom error messages is a plus, so you can add more information in a message.
In case above it is clear that floating IP is not is a list (Also we can add server name, and other details).

From `hamcrest .assert_that`, in case of error, we can get only that some IP not in the list of other.